package.json

```diff
@@ -13,10 +13,13 @@
     "build": "vite build src/auth_client_demo_assets/vanilla --config vite.config.js",
     "prebuild": "npm run generate",
     "start": "vite src/auth_client_demo_assets/vanilla --config vite.config.js",
+    "start:dfx": "dfx start --background --clean --host \"127.0.0.1:4943\"",
     "start:react": "vite src/auth_client_demo_assets/react --config vite.config.react.js",
```

Comment:
react?

Reply:
yeah, I have a bunch of frameworks side-by-side here

Reply:
Ah it's in the root, my bad. Coolio